.github/workflows/phpunit-tests.yml

```diff
@@ -112,45 +112,39 @@ jobs:
           node-version-file: '.nvmrc'
           cache: npm
 
+      ##
+      # This allows Composer dependencies to be installed using a single step.
+      #
+      # Since the tests are currently run within the Docker containers where the PHP version varies,
+      # the same PHP version needs to be configured for the action runner machine so that the correct
+      # dependency versions are installed and cached.
+      ##
+      - name: Set up PHP
+        uses: shivammathur/setup-php@1a18b2267f80291a81ca1d33e7c851fe09e7dfc4 # v2.22.0
+        with:
+          php-version: '${{ matrix.php }}'
+          coverage: none
+
+      # Since Composer dependencies are installed using `composer update` and no lock file is in version control,
+      # passing a custom cache suffix ensures that the cache is flushed at least once per week.
+      - name: Install Composer dependencies
+        uses: ramsey/composer-install@83af392bf5f031813d25e6fe4cd626cdba9a2df6 # v2.2.0
+        with:
+          custom-cache-suffix: $(/bin/date -u --date='last Mon' "+%F")
+          composer-options: "${{ matrix.php == '8.2' && '--ignore-platform-req=php+' || '' }}"
```

Comment:
`--ignore-platform-req=php+` is no longer needed for PHP 8.2 AFAICS, so the whole `composer-options` line can be removed.